Pipfile

```diff
@@ -32,6 +32,9 @@ graphql-core = ">=2.1,<3"
 pytz = ">=2017.2"
 six = ">=1.10.0"
 sqlalchemy = ">=1.3.0,<2"
+neo4j = "==1.7.4"
```

Comment:
[question](https://github.com/kensho-technologies/graphql-compiler/pull/369/files#r304106727) from #369: 

> General question: I notice that some of these other ones don't specify a particular version but rather a set of possible versions. How big a deal is this? I remember being told in some of the Github issues that some functionality that we rely on (e.g. the output format for RedisGraph) would change in future releases.

Reply:
This Pipfile is only used for the test setup, and does not influence what packages are considered compatible when actually installing the compiler package itself. That said, when adding a requirement here, you should:
- Consider what versions of the dependencies we'd like to support. For example, we generally want to support the version that was latest at the time of writing, and any minor/patch releases around it, but not a major version bump (first number) since that might contain breaking changes.
- Then, if the package is required when using the compiler, and not just while testing it, you should make sure that the `setup.py` of the compiler package has the correct requirement that you have decided on. If the package is only used in tests, skip this step.
- Finally, you should add the package to the Pipfile, in the `dev` section if only used in tests, and in the non-dev section otherwise, and make sure the version bound matches what you decided earlier.

Reply:
So specifically for this line in particular, I'd make it `neo4j = ">=1.7.4,<2".